fastlane/spec/actions_specs/set_github_release_spec.rb

```diff
@@ -4,7 +4,7 @@
       it "sends and receives the right content from GitHub" do
         stub_request(:post, "https://api.github.com/repos/czechboy0/czechboy0.github.io/releases").
           with(body: "{\"tag_name\":\"tag33\",\"name\":\"Awesome Release\",\"body\":\"Bunch of new things :+1:\",\"draft\":false,\"prerelease\":false,\"target_commitish\":\"test\"}",
-          headers: { 'Authorization' => 'Basic MTIzNDVhYmNkZQ==', 'Host' => 'api.github.com:443', 'User-Agent' => 'fastlane-set_github_release' }).
+          headers: { 'Authorization' => 'Basic MTIzNDVhYmNkZQ==', 'Host' => 'api.github.com:443', 'User-Agent' => 'fastlane-github_api' }).
```

Comment:
Great idea, I wasn't sure if it would be a problem - I assumed it was just because it was a required attribute for the Github API, and not for any other reason, see: https://developer.github.com/v3/#user-agent-required

But I can add an option if people will actually use it?